### DISCUSSION TITLE
Duda sobre subir repositorio desde CMD

### DISCUSSION THREAD

**Discussion by DiegoPenalozaJuarez** [LaunchX-InnovaccionVirtual/CursoIntroPython#242] [Q&A] [answered]
Hola

Espero alguien pueda ayudarme con la falla que estoy teniendo a la hora de subir mis katas al repositorio en github

Al momento de realizar el siguiente comando: 
`git commit -m "first commit"`

me marca el siguiente mensaje:
![image](https://user-images.githubusercontent.com/99508394/153937349-3a932129-8e87-4680-bf81-b763fc863d4a.png)

Alguien sabe como resolverlo? Me dice Author identity unknown  ***Please tell me who you are.

Gracias por su ayuda.

**Comment by JavierMendez-Coder** [comment, marked as answer]
Buenas tardes. Lo que sucede es que tienes que agregar tu nombre y correo electronico ya que git utiliza esos datos para guardar al autor del commit. Para realizarlo, puedes utilizar los comandos que te muestra:
```cmd
git config --global user.email "tu@ejemplo.com"
git config --global user.name "Tu nombre"
```
Espero haberte ayudado.

**Reply by DiegoPenalozaJuarez** [reply]
Hola @JavierMendez-Coder gracias por responder. Osea deben ser el correo electronico con el cual me registre en github? y me imagino igual mi usuario de git.

**Reply by JavierMendez-Coder** [reply]
Por `user.name` es tu nombre normal (Javier Méndez, en mi caso) y en `user.email` preferiblemente el correo electrónico de tu cuenta de github, pero realmente puede ser cualquiera. Ya que a la hora de hacer el push al repositorio remoto te pide que inicies sesión en la cuenta del mismo para que lo autorices.

**Reply by DiegoPenalozaJuarez** [reply]
Listo ya quedo muchas gracias compañero fuiste de mucha ayuda

**Reply by RodolfoMorquecho** [reply]
Disculpen, cuando hago el ultimo paso en el git bash, cuando ingreso el git push me manda a una pagina de github para autorizar una credencial manager, debo autorizar o a ustedes no les aparecío.

**Reply by JavierMendez-Coder** [reply]
Sip, tienes que autorizarlo. Saludos 👋.

**Comment by ascamarena** [comment]
Puedes darle acceso a tus credenciales. Solo pedirá una autenticación de 2 factores. Suerte.
https://github.com/GitCredentialManager/git-credential-manager

**Reply by DiegoPenalozaJuarez** [reply]
Lo intentare con esa autenticacion, gracias
